extensions/amp-orientation-observer/0.1/test/test-amp-orientation-observer.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {AmpOrientationObserver} from '../amp-orientation-observer';
+
+describes.sandboxed('amp-orientation-observer', {}, env => {
+  let impl;
+  let triggerEventSpy;
+
+  function init() {
+    const elem = {
+      getAttribute(attr) {
+        if (attr == 'alpha-range') {
+          return '0 360';
+        }
+        if (attr == 'beta-range') {
+          return '-180 180';
+        }
+        if (attr == 'gamma-range') {
+          return '-90 90';
+        }
+      },
+    };
+    elem.ownerDocument = {
+      defaultView: window,
```

Comment:
Prefer to reference `env.win`. `env.win` is the test environment's window object, where `window` is the outer window used by Mocha.

Reply:
Done